.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.5
```

Comment:
this file shouldn't be saved to the repo

Reply:
Apologies, this is a new machine and I hadn't set up the `.gitignore` file yet. I'll remove it!